src/src/Helpers/Networking.cpp

```diff
@@ -1544,7 +1544,26 @@ int http_authenticate(const String& logIdentifier,
     event += '=';
     event += httpCode;
     eventQueue.addMove(std::move(event));
+    #if FEATURE_THINGSPEAK_EVENT
+      // Generate event with the response of a 
+      // one value one channel tingspeak request
+      // e.g. (sendToHTTP,api.thingspeak.com,80,/channels/143789/fields/5/last)
+      // where first eventvalue is the channel number, the second the field number
+      // and the third is the value received by the request
+      if (equals(host, F("api.thingspeak.com")) && equals(uri.substring(uri.length() - 5, uri.length()), F("/last")) && httpCode == 200) {
+      String revent = F("reply#");
+      revent += host;
+      revent += '=';
+      revent += uri.substring(0, uri.length() - 14).substring(10, uri.length()); //get the channel number
+      revent += ',';
+      revent += uri.substring(uri.length() - 6, uri.length() - 5); //get the field number
+      revent += ',';
+      revent += http.getString().substring(0, 21);
```

Comment:
Can you give some example(s) in comments about the received reply and the resulting event?

Reply:
I can...